### DISCUSSION TITLE
Build outside of root package.json

### DISCUSSION THREAD

**Discussion by cupcakearmy** [parcel-bundler/parcel#5069] [General]
So basic question:

I need to build some files outside the project folder in a os tmp folder. If i build the files inside of the folder it's no problem, outside, parcel starts to install deps, and failing afterwards.
Any help would be much appreciated!!

**Comment by mischnic** [comment]
Please provide an example. Which dependencies? Are you using `--dist-dir`?

**Comment by cupcakearmy** [comment]
@mischnic  thanks for the quick reply.
This is the minimal setup to replicate

```
..
├── project
│   ├── package.json
│   ├── src
│   │   ├── index.html
│   │   └── main.js
│   └── yarn.lock
└── src
    ├── index.html
    └── main.js
```

The 2 `src` folders are identical. 

```
// index.html
<!DOCTYPE html>
<html>
  <body>
    <script src="./main.js"></script>
  </body>
</html>
```

```
// main.js
import dayjs from 'dayjs'

console.log(new dayjs().toString())
```

```
// package.json
{
  "dependencies": {
    "dayjs": "^1.8.34"
  },
  "devDependencies": {
    "parcel": "^2.0.0-nightly.380"
  }
}
```



now if (inside of the project folder) i run:

`yarn parcel build ./src/index.html` -> works

`yarn parcel build ../src/index.html` -> breaks

```
🚨 Build failed.
Error: Cannot find module '@babel/core' from '*****/src'
```

**Reply by cupcakearmy** [reply]
I would guess this is related https://github.com/parcel-bundler/parcel/issues/2318

I've tried both the beta and nightly of v2 and also v1
